advocacy_docs/partner_docs/VisionXGuide/04-ConfiguringTheIntegration.mdx

```diff
@@ -0,0 +1,18 @@
+---
+title: 'Configuring the Integration'
+description: 'Walkthrough on configuring VisionX'
+---
+
+Implementing VisionX with EDB Postgres Advanced Server and EDB Postgres Extended requires the following components:
+
+- EDB Postgres Advanced Server
+- EDB Postgres Extended
+- VisionX software
+- Oracle
```

Comment:
What specific Oracle item are we talking about?  Also would this only be if you are migrating forms from Oracle? 
If you are just running VisionX with EDB you don't need Oracle correct?